bimcollab_zoom/com.github.wycomco.bimcollab_zoom.download.recipe

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Download recipe for the latest version of Kubus' BIMcollab ZOOM.</string>
+	<key>Identifier</key>
+	<string>com.github.wycomco.bimcollab_zoom.download</string>
+	<key>Input</key>
+	<dict/>
+	<key>MiniumumVersion</key>
+	<string>0.2.9</string>
+	<key>Process</key>
+	<array>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>re_pattern</key>
+				<string>https://www\.kubusinfo\.nl/downloads/BIMcollab ZOOM\D.*\.dmg</string>
+				<key>result_output_var_name</key>
+				<string>url</string>
+				<key>url</key>
+				<string>https://support.bimcollab.com/de/Support/Support/Downloads/BIMcollab-ZOOM</string>
+			</dict>
+			<key>Processor</key>
+			<string>URLTextSearcher</string>
+		</dict>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>find</key>
+				<string> </string>
+				<key>input_string</key>
+				<string>%url%</string>
+				<key>replace</key>
+				<string>%20</string>
+			</dict>
+			<key>Processor</key>
+			<string>com.github.homebysix.FindAndReplace/FindAndReplace</string>
```

Comment:
Is there any requirement to reference processors which are not included by default? Or is it up to the user to identify the requirements when first running the recipe?